python-venv/cmstools/lib/common/log.py

```diff
@@ -30,7 +30,7 @@
 import os
 import sys
 
-from barebones_image_test.defs import BB_TEST_TIMESTAMP
+from cmstools.lib.common.defs import BB_TEST_TIMESTAMP
```

Comment:
`BB_TEST_TIMESTAMP` sounds specific to the bare bones test. If so, we should probably move that definition back into the BB module somewhere. But since this is the common module, I assume this is a generic timestamp that you plan on using in other places. I'd rename the definition in the other file (and then to avoid mass changes inside the BB test itself you could still do `from whatever import x as BB_TEST_TIMESTAMP`)